rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -11,4 +11,13 @@ if [ -z `getent passwd spinnaker` ]; then
   useradd --gid spinnaker spinnaker -m --home-dir /home/spinnaker
 fi
 
+# install packer
+pushd
+mkdir /packer && \
```

Comment:
I don't think `/packer` is the right way to do this. I'd create a tempdir and install there, removing the tempdir after.